templates/shared/searchfile.tmpl

```diff
@@ -0,0 +1,14 @@
+<div class="file-body file-code code-view">
+	<table>
+		<tbody>
+			{{range .result.Lines}}
+				<tr>
+					<td class="lines-num">
+						<a href="{{$.repolink}}/src/commit/{{PathEscape $.result.CommitID}}/{{PathEscapeSegments $.result.Filename}}#L{{.Num}}"><span>{{.Num}}</span></a>
+					</td>
+					<td class="lines-code chroma"><code class="code-inner">{{.FormattedContent}}</code></td>
```

Comment:
When you render `string` into HTML, it will be escaped. So, isn't it causing double-escaping?

Reply:
Yes, it happened.
![image](https://github.com/go-gitea/gitea/assets/18380374/0fe78501-401c-4487-a2b2-0d3b31c20f4b)
![image](https://github.com/go-gitea/gitea/assets/18380374/3ed1d591-12c4-47a5-98fa-d42bf3a99f4d)
Need to convert it back to HTML. Now I understand why it was `template.HTML` 👍